ydb/core/tx/schemeshard/schemeshard_info_types.h

```diff
@@ -2391,8 +2396,20 @@ struct TTableIndexInfo : public TSimpleRefCount<TTableIndexInfo> {
         return result;
     }
 
+    TString DescriptionToStr() const {
```

Comment:
`ToStr` has a misleading connotation. According to our codebase it doesn't have a meaning of "encode value to a sequence of bytes" but rather "make something printable to be viewed by a person".
I would recommend using something more accurate: `SerializedDescription` or something.

Reply:
Ok, changed to `SerializeDescription`, not `SerializedDescription` because I want it to be clear that it's not just getter